mmdet/datasets/transforms/loading.py

```diff
@@ -255,8 +260,8 @@ def _load_bboxes(self, results: dict) -> None:
         for instance in results['instances']:
             gt_bboxes.append(instance['bbox'])
             gt_ignore_flags.append(instance['ignore_flag'])
-        results['gt_bboxes'] = np.array(
```

Comment:
I think this is a little bit strange:
1. others are np.ndarry, but gt_bboxes is a boxlist (Tensor). 2. every time process the boxes, you should convert boxlist to numpy and reconvert back to boxlist